### ISSUE TITLE
Cannot get any spotify to work.

### ISSUE THREAD

**Issue by gazza7** [hankhank10/vinylemulator#12]
Hi,

I've installed by following each step and every thing is working until I get to the point of playing on Spotify.
it doesn't matter if its a album or playlist all I get is a status error.
I have no problem with using http://192.x.x.x:5005/Dining%20Room/playpause  volume, zone etc.

Its only when I try with Spotify eg http://192.x.x.x:5005/Dining%20Room/spotify:user:spotify:playlist:5huIma0YU4iusLcT2reVEU
or http://192.168.1.4:5005/Dining%20Room/spotify:user:spotify:album:0PMasrHdpaoIRuHuhHp72O

I've tried numerous albums and playlist but same outcome.

Regards

G smith

**Comment by hankhank10** [comment]
Do you get an error message?

Have you confirmed that spotify plays happily from your Sonos app?

**Comment by gazza7** [comment]
Hi,

Yes the the Sonos  app works on my phone and of my desktop computer, but they do bot bring my playlist though, only the playlist made for you from Spotify eg Daily Mix and Music for you on Thursday etc.

The only message I get is Status Error.

I am going to reinstall to see if this changes anything.

Regards

**Comment by gazza7** [comment]
After reinstalling and using different node version, I'm still getting the 'status error'
![image](https://user-images.githubusercontent.com/3785032/90136201-b53af300-dd6b-11ea-9e96-b7b867dab6f9.png)

**Comment by hankhank10** [comment]
This is an error with the node-sonos-http-api so you will have better luck
looking for support at https://github.com/jishi/node-sonos-http-api

**Comment by gazza7** [comment]
Hi,
After reinstalling the Raspi and the additional software, it still would not play Spotify with the API.
So I had  a rest from it for a while and I notice that some adverts was playing on Spotify via the Sonos app.
So I deactivated my Spotify account and re-added, but I was still getting adverts every half hour, even though its a premium account.  So I decided to reset my Sonos connect and reinstall the app.
After everything was back up and running I re-added my Spotify account, but still no luck at playing Spotify via the API.  I then notice the adverts had stopped and all my playlist and albums had populated into the Sonus app. I then  tried playing a Spotify playlist via the API, but still silence, then I notice the web page showing {"status":"success"} then suddenly the music started blasting out.  So it looks like the Sonus app and Sonus connect wasn't playing ball with each other. Two days off fun now over.

Regards
G
